CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Unreleased
+
+* **Overall**
+
+* **Metrics**
+  * [FIX] Updated Go pod monitors to not scan unnecessary ports or pods.
```

Comment:
Perhaps: [FIX] Updated Go podMonitors to eliminate scans of unnecessary ports or pods.